src/main/resources/application.yml

```
@@ -0,0 +1,17 @@
+spring:
+  profiles:
+    active: develop
+---
+
+spring:
+  config:
+    activate:
+      on-profile: production
+    import: sub/application-db.yml
```

submodule 저장 디렉토리 이름 secrets 로 되어있어요
수정 부탁드려요
```suggestion
    import: secrets/application-db.yml
```